packages/account/src/Components/poi/idv-status/idv-failed/__tests__/idv-failed.spec.tsx

```diff
@@ -26,9 +26,21 @@ jest.mock('@deriv/shared', () => ({
 
 jest.mock('Components/forms/personal-details-form', () => jest.fn(() => <div>PersonalDetailsForm</div>));
 jest.mock('Components/forms/idv-form', () => jest.fn(() => <div>IDVForm</div>));
-jest.mock('Assets/ic-poi-name-example.svg', () => jest.fn(() => 'PoiNameExample'));
-jest.mock('Assets/ic-poi-dob-example.svg', () => jest.fn(() => 'PoiDobExample'));
-jest.mock('Assets/ic-poi-name-dob-example.svg', () => jest.fn(() => 'PoiNameDobExample'));
+
+jest.mock('@deriv/quill-icons', () => ({
+    ...jest.requireActual('@deriv/quill-icons'),
+    DerivLightNamePoiIcon: () => 'DerivLightNamePoiIcon',
+}));
+
+jest.mock('@deriv/quill-icons', () => ({
+    ...jest.requireActual('@deriv/quill-icons'),
+    DerivLightNameDobPoiIcon: () => 'DerivLightNameDobPoiIcon',
+}));
+
+jest.mock('@deriv/quill-icons', () => ({
+    ...jest.requireActual('@deriv/quill-icons'),
+    DerivLightDobPoiIcon: () => 'DerivLightDobPoiIcon',
+}));
```

Comment:
```suggestion

jest.mock('@deriv/quill-icons', () => ({
    ...jest.requireActual('@deriv/quill-icons'),
    DerivLightNamePoiIcon: () => 'DerivLightNamePoiIcon',
    DerivLightNameDobPoiIcon: () => 'DerivLightNameDobPoiIcon',
    DerivLightDobPoiIcon: () => 'DerivLightDobPoiIcon',
}));
```
Instead of adding 3 mock separately, you can put all three in single mock.